deal.II-toolchain/packages/petsc.package

```diff
@@ -2,8 +2,11 @@
 ## PETSc                                                                      ##
 ################################################################################
 
-VERSION=3.19.0
-CHECKSUM=3d8820670f0d67aa04ce41b91f8eae62
+VERSION=3.19.3
```

Comment:
Can you update to 3.19.4? I think it just got released.

Reply:
Absolutely!